src/field_theory/adjoin.lean

```diff
@@ -326,9 +357,6 @@ lemma subsingleton_of_findim_adjoin_eq_one (h : ∀ x : E, findim F F⟮x⟯ = 1
   subsingleton (intermediate_field F E) :=
 subsingleton_of_bot_eq_top (bot_eq_top_of_findim_adjoin_eq_one h)
 
-instance [finite_dimensional F E] (K : intermediate_field F E) : finite_dimensional F K :=
-  finite_dimensional.finite_dimensional_submodule (K.to_subalgebra.to_submodule)
```

Comment:
Just a quick comment about this removal: It already occurs in intermediate_field.lean as finite_dimensional_left

Reply:
Maybe `finite_dimensional_left` should even be defined for `is_scalar_tower`, but I'll leave that to another PR.